docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -658,6 +666,66 @@ Working directory inside the container.
 
 The default working directory for running binaries within a container is the root directory (/). The image developer can set a different default with the WORKDIR instruction. This option overrides the working directory by using the -w option.
 
+## Pod units [Pod]
```

Comment:
It would be nice to have an example in the man pages on how to tie a `.pod` and a `.container` unit together.

Reply:
Added an example at the Examples section